inst/emails/contact_one_month_remaining

```diff
@@ -4,13 +4,13 @@ The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would li
 
 NSF requires that all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive before the end of your award. Additionally, data and derived data products are often required to be submitted on this same timeframe (although the timing and submission requirements for data varies between programs). More detailed information about what will be expected of you can be found at https://arcticdata.io/submit/#who-must-submit.
 
-If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
+See the reporting guidelines{https://www.nsf.gov/pubs/2016/nsf16055/nsf16055.jsp} from NSF for details, but generally NSF requires that URL's for archived metadata and data should be included in these reports in the section entitled "Products-Websites.".
 
-Please be sure to submit your data and metadata to the Arctic Data Center sufficiently before your report is due to allow adequate processing time for us to review your submission and for any needed revisions to be prepared.  While we strive to process data as quickly as possible, larger data sets can take significantly more time to process, so please plan accordingly.  Also, there will be less need for multiple rounds of revisions if your initial submission uses open, non-proprietary data formats, includes data that is well-organized, and includes high quality, detailed documentation.  For example, you should document all aspects of your data, including detailed methodological descriptions and structural information about the contents of your data.  See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations.
+If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
 
-See the reporting guidelines{https://www.nsf.gov/pubs/2016/nsf16055/nsf16055.jsp} from NSF for details, but generally NSF requires that URL's for archived metadata and data should be included in these reports in the section entitled "Products-Websites.".
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. We advise that this is taken to account when planning a submission to the Arctic Data Center for your report.
```

Comment:
Do we want to mention that, while many data sets can be processed in a day or two, larger or more complex data can sometimes take a couple of weeks to process?  I think giving people an idea of the time scale would be helpful in this and other emails above.